grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
That also happens to be why I haven't finished reviewing the XML PR.  This probably won't make it into either the linguist or Atom releases as I'll need time to reflect and think about your questions.

And also, would it be feasible to include all of source.js in here after https://github.com/atom/first-mate/pull/90 is merged?